### ISSUE TITLE
Edit saved server

### ISSUE THREAD

**Issue by teal77** [teal77/trireme#8]
There should be a way to edit a saved server. The screen in settings / server details should let the user edit the server details. Right now they have to delete and then add the server again with the updated details.

Original #5

**Comment by lepigocher** [comment]
Hello,

Could you please explain how to delete a server? I have been able to create a new one but not to delete the server with wrong settings.

Thanks!

**Comment by teal77** [comment]
Settings > View/Remove server > Tap on the server you wish to delete > Tap on Trash can icon in toolbar.

It was added in 0.9.1. So its not yet available in FDroid due to build script errors. It's being worked on.

**Comment by lucidnx** [comment]
I would also like ability to remove all servers. Now it just throw message saying I could not remove last server/connection from app.

**Comment by teal77** [comment]
Agree. App was written in a way that having no servers added is only an edge case and something which only happens on new install. It should be treated like a valid state.
